ERCS/ERC-XXXX.md

```diff
@@ -0,0 +1,130 @@
+---
+eip: XXXX
+title: Minimal Wallet-Managed Auto-Login for SIWE
+description: Defines a wallet-local allowlist for automatic signing of EIP-4361 messages when simple, deterministic match rules succeed. Policies are created and managed only by the wallet/user.
+author: Ivo Georgiev <ivo@ambire.com>, Vijay Krishnavanshi <vijay@fileverse.io>
+discussions-to: https://ethereum-magicians.org/t/wallet-managed-auto-login-for-siwe-minimal/XXXX
```

Comment:
Please create a discussions topic in Eth Magicians linking to this PR
https://ethereum-magicians.org/c/ercs/57